README.md

```diff
@@ -6,195 +6,252 @@
     <rect x="80" y="40" rx="3" ry="3" width="250" height="10" />
   </ContentLoader>
 )
 ```
 
+**Any doubt yet?** Take a look at [codesandbox.io](https://codesandbox.io/s/moojk887z9) working.
+
 ## Options
 
-| Name                    | Type        | Default                | Description                                                               |
-| ----------------------- | ----------- | ---------------------- | ------------------------------------------------------------------------- |
-| **animate**             | `{Boolean}` | `true`                 | `false` to render with no animation                                       |
-| **ariaLabel**           | `{String}`  | `Loading interface...` | Describe what element it is                                               |
-| **speed**               | `{Number}`  | `2`                    | Animation speed in seconds                                                |
-| **className**           | `{String}`  | `''`                   | Classname in the `<svg/>`                                                 |
-| **width**               | `{Number}`  | `400`                  | viewBox width of `<svg/>`                                                 |
-| **height**              | `{Number}`  | `130`                  | viewBox height of `<svg/>`                                                |
-| **rtl**                 | `{Boolean}` | `false`                | Right-to-left animation                                                   |
-| **preserveAspectRatio** | `{String}`  | `xMidYMid meet`        | Aspect ratio option of `<svg/>`                                           |
-| **primaryColor**        | `{String}`  | `#f3f3f3`              | Background                                                                |
-| **secondaryColor**      | `{String}`  | `#ecebeb`              | Animation color                                                           |
-| **primaryOpacity**      | `{Number}`  | `1`                    | Background opacity (0 = transparent, 1 = opaque)                          |
-| **secondaryOpacity**    | `{Number}`  | `1`                    | Animation opacity (0 = transparent, 1 = opaque)                           |
-| **style**               | `{Object}`  | `null`                 | Ex: `{ width: '100%', height: '70px' }`                                   |
-| **uniquekey**           | `{String}`  | random unique id       | Use the same value of prop key, that will solve inconsistency on the SSR. |
-
-### Examples
-
-#### Facebook Style
+**`animate?: boolean`**
+
+Default is `true`. To render with no animation, set `false`.
+
+**`ariaLabel? string | boolean`**
+
+Default is `Loading interface...`. It's used to describe what element it is. Use `false` to remove.
+
+**`speed?: number`**
+
+Default is `2`. Animation speed in seconds.
+
+**`className? string`**
+
+Default is an empty string and the classname will set in the `<svg />` element.
```

Comment:
Defaults to an empty string. The classname will be set in the `<svg />` element.